Packs/CortexAttackSurfaceManagement/.pack-ignore

```diff
@@ -3,3 +3,6 @@ ignore=IF100
 
 [known_words]
 ASM
+
+[file:Cortex_ASM_-_ASM_Alert.yml]
+ignore=PB121
```

Comment:
@johnnywilkes Why is it giving error PB121?

Reply:
@idovandijk  this is what comes up on validate
![image](https://user-images.githubusercontent.com/32227961/202443217-d903d0e2-f4fc-437f-bada-e35c7f4c5190.png)

My understanding is that this is coming up because it thinks "incident.do?sysparm_query=number=" is a variable when it is actually a string literal (used to created to full ServiceNow URL).  Therefore I ignored the error as a false positive.  Make sense?

![2022-11-17_07-13-58](https://user-images.githubusercontent.com/32227961/202443536-6ec7959c-f434-4f3f-ba3c-5961175d00fe.png)

Reply:
Got it. That's indeed a false positive :)